client/src/components/Login/index.jsx

```diff
@@ -2,12 +2,35 @@ import React, { Component } from 'react';
 import './style.css';
 import Logo from '../Logo';
 
+
 export default class extends Component {
   state = {
     email: '',
     password: '',
+    msg:''
   };
 
+   
+  onSubmit = event => {
+    event.preventDefault();
+    console.log(event.target.email.value);
+    const requestInfo = {
+      emailValue: event.target.email.value,
+
+      passwordValue: event.target.password.value
+    }
+    fetch('/login', {
+      method: 'POST',
+      body: JSON.stringify(requestInfo),
+      headers: {
+        'content-type': 'application/json'
+      }
+    })
+    .then(res=>res.json())
+    .then(({msg}) => {
+     this.setState({msg:'login successfully'});
```

Comment:
You could redirect the user to a different page here (after a short timeout) if you'd like.
You can use react-router-dom's [withRouter](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/docs/api/withRouter.md) to have this.props.history injected into your component, and then you could call `this.props.history.push('/home')` or something like that.